src/client.rs

```diff
@@ -125,6 +125,26 @@ impl<'a> Client<'a> {
         .into_index(self))
     }
 
+    /// Delete an index from its UID if it exists.
+    /// To delete an index if it exists from the [index object](../indexes/struct.Index.html), use
+    /// [the delete_if_exists method](../indexes/struct.Index.html#method.delete_if_exists).
```

Comment:
It's better to [link to items by name](https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html).

```suggestion
    /// To delete an index if it exists from the [`Index`] object, use the [Index::delete_if_exists] method.
```